src/vuln_analysis/server/webhooks/config.py

```
@@ -107,7 +107,13 @@ class WebhookConfig:
     # SQLite ledger path for restart-durable idempotency. Defaults under the reports root.
     ledger_path: str | None = None
 
-    def __post_init__(self):
+    def __post_init__(self) -> None:
+        if not math.isfinite(self.poll_interval_seconds) or self.poll_interval_seconds <= 0:
+            raise ValueError(f"{_ENV_PREFIX}POLL_INTERVAL_SECONDS must be finite and positive")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)